### PR TITLE
geolocation: update logging of redirect path for better debugability

### PR DESCRIPTION
**Pull request by emranemran**

### CODE REVIEW

File: handlers/geolocation/geolocation.go [hunk 1]
```diff
@@ -156,13 +156,20 @@ func (c *GeolocationHandlersCollection) RedirectHandler() httprouter.Handle {
...
-			"lon":           lon,
+			"redirect-type":      redirectType,
+			"playbackID":         playbackID,
+			"from":               r.URL.String(),
+			"dns-chosen-region":  c.Config.OwnRegion,
+			"mist-chosen-region": bestNode,
+			"lat":                lat,
+			"lon":                lon,
 		})
 		glog.Infof(string(jsonRedirectInfo))
```
Comment (pwilczynskiclearcode) [inline handlers/geolocation/geolocation.go:174]: Could we also produce a prometheus metric in here? We're missing redirect data on Grafana. 
```go

   metrics.Metrics.RedirectBestNode.WithLabelValues(redirectType, playbackID, r.URL.string(), c.Config.OwnRegion, bestNode).Inc()
```

Comment (pwilczynskiclearcode) [inline handlers/geolocation/geolocation.go:174]: Could we also produce a prometheus metric in here? We're missing redirect data on Grafana. 
```go
metrics.Metrics.RedirectBestNode.WithLabelValues(redirectType, playbackID, r.URL.string(), c.Config.OwnRegion, bestNode).Inc()
```

File: handlers/geolocation/geolocation.go [hunk 2]
```diff
@@ -156,13 +156,20 @@ func (c *GeolocationHandlersCollection) RedirectHandler() httprouter.Handle {
 			w.WriteHeader(http.StatusInternalServerError)
 			return
 		}
+		var redirectType string
+		if isStudioReq {
+			redirectType = "ingest"
+		} else {
+			redirectType = "playback"
+		}
```
Comment (thomshutt) [inline handlers/geolocation/geolocation.go:164]: [](url)
```suggestion
		var redirectType = "playback"
		if isStudioReq {
			redirectType = "ingest"
		}
```

File: handlers/geolocation/geolocation.go (hunk 1 above)
Comment (thomshutt) [reply handlers/geolocation/geolocation.go:174]: @emranemran I'm going to merge and get this deployed, but could you do a followup PR to address Pawel's suggestion please

File: handlers/geolocation/geolocation.go [hunk 3]
```diff
@@ -156,13 +156,18 @@ func (c *GeolocationHandlersCollection) RedirectHandler() httprouter.Handle {
...
-			"to":            bestNode,
-			"lat":           lat,
-			"lon":           lon,
+			"redirect-type":      redirectType,
+			"playbackID":         playbackID,
+			"from":               r.URL.String(),
+			"dns-chosen-region":  c.Config.OwnRegion,
+			"mist-chosen-region": bestNode,
+			"lat":                lat,
+			"lon":                lon,
```
Comment (thomshutt) [inline handlers/geolocation/geolocation.go:170]:
```suggestion
			"redirectType":      redirectType,
			"playbackID":         playbackID,
			"from":               r.URL.String(),
			"dnsChosenRegion":  c.Config.OwnRegion,
			"mistChosenRegion": bestNode,
			"lat":                lat,
			"lon":                lon,
```

**Review by thomshutt — APPROVED** [review]

**Pull request merged**